test/Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -1302,7 +1302,7 @@ public async Task ServerCanAbortConnectionAfterUnobservedClose(ListenOptions lis
                 await appFuncCompleted.Task.DefaultTimeout();
             }
 
-            mockKestrelTrace.Verify(t => t.ConnectionStop(It.IsAny<string>()), Times.Once());
+            mockKestrelTrace.Verify(t => t.ConnectionStop(It.IsAny<string>()), Times.AtMostOnce());
```

Comment:
Is there a point in checking these at all if we allow 0 and 1? Maybe we should just remove them entirely. I suppose we are making sure ConnectionStop is never called more than once still.

Reply:
There's almost no point other than to remind us that there was once a useful assertion here. Be extra careful not to apply these changes to release/2.2 or master when we merge.